opennms-doc/guide-admin/src/asciidoc/index.adoc

```diff
@@ -213,4 +214,4 @@ include::text/plugins/opennms-es-rest.adoc[]
 
 [[ga-special-cases-and-workarounds]]
 == Special Cases and Workarounds
-include::text/workarounds/snmp.adoc[]
\ No newline at end of file
+include::text/workarounds/snmp.adoc[]
```

Comment:
I think previously it was missing a carriage return at the EOF and his edit added one, or vice-versa.